### DISCUSSION TITLE
New device alert.

### DISCUSSION THREAD

**Discussion by juanep97** [diezo/Ensta#55] [General]
After logging using 'Host(username, password)' I receive an alert saying I logged with a new device:
```
Device · Python Requests
```
I think this unnecessarily reveals some information to Instagram, and it should by default send a fake user agent such as Firefox or Google Chrome, to reduce the risk of the account being banned.

**Comment by diezo** [comment]
@juanep97 do you have any possible solutions to this issue?

**Comment by juanep97** [comment]
I see in the code that it's sending `sech-ch-ua` but not `User-Agent`, might have to do with that?

**Comment by diezo** [comment]
i've fixed this issue in the latest release. please update ensta using: ```pip install ensta --upgrade```.
